packages/shared/utils.ts

```diff
@@ -33,3 +33,8 @@ export function merge(target: any, source: any) {
 export function isIndex(value: unknown): value is number {
   return Number(value) >= 0;
 }
+
+export function toNumber(value: string): number | string {
```

Comment:
Got it from `@vue/shared`. Do you think worth adding `@vue/shared` as dependency?

Reply:
I prefer not to add any dependencies at this time, so it's fine at the moment 👍